0_global_functions.R

```diff
@@ -393,6 +393,9 @@ write_log <- function(msg, file_path = log_path, ...) {
 }
 
 convert_quarter_to_year <- function(quarter_string){
+  # see function definition for more info, but this give the user a useful
+  # error if length > 1 (grouped portfolio from multiple timestamps)
```

Comment:
Nit: If anywhere I would move this comment to the function's body. Here the risk is high that the comment will fall out of sync with the implementation.

Reply:
this is in the function body, but I clarified which function I'm referring to.